test/testlib/OpenSSL/Test.pm

```diff
@@ -937,45 +1025,25 @@ sub __fixup_cmd {
     return undef;
 }
 
-sub __build_cmd {
+# __decorate_cmd NUM, CMDARRAYREF
+#
+# __decorate_cmd takes a command number NUM and a command token array
+# CMDARRAYREF, builds up a command string from them and decorates it
```

Comment:
It seems to add quite a bit of complexity AFAICT. Various functions return a "sub", with the only reason so that they can later take a num (I think)...but then the num is never used.